src/lazyload.js

```diff
@@ -241,6 +245,11 @@
             elementsLength = (!elements) ? 0 : elements.length,
             processedIndexes = [];
 
+        /* Calling _loopThrough callback */
+        if (settings.callback_loopThrough) {
+            settings.callback_loopThrough(_isFirstLoop);
```

Comment:
the _isFirstLoop variable cannot be a local variable shared with all instances of lazyload, it must be an instance variable instead. So it should be referenced as this._isFirstLoop here.